<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quicklaunchConfig</class>
 <widget class="QWidget" name="quicklaunchConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure QuickLaunch</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Visible icons:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="iconSizeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Icon size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="iconSizeSpin">
     <property name="suffix">
      <string>px</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="icons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="specialValueText">
      <string>Unlimited</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="dialogIconSizeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dialog icon size:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="dialogIconSizeSpin">
     <property name="suffix">
      <string>px</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>iconSizeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>iconSizeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconSizeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>iconSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogIconSizeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>dialogIconSizeSpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogIconSizeSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dialogIconSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
